e2etests/testcase_test.go

```diff
@@ -3696,7 +3696,7 @@ Bug Fix(es) and Enhancement(s):
 				NamespaceName: "rhel:8",
 				VersionFormat: "rpm",
 				Version:       "6.0.6-1.el8_6.x86_64",
-				FixedBy:       "6.0.32-1.el8_10",
+				FixedBy:       "6.0.35-1.el8_10",
```

Comment:
not here, but in the rhel 8 cups-filters test, can you check for [RHSA-2024:7463](https://access.redhat.com/errata/RHSA-2024:7463), too?

Reply:
![image](https://github.com/user-attachments/assets/c94fce4f-a585-49a3-bec5-612c8bb4a1f8)

Test already there?

Reply:
ah they added this CVE in after the fact. Cool